pom.xml

```diff
@@ -245,11 +245,24 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>4.2.1.RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+
         <!-- Testing JSON -->
         <dependency>
             <groupId>com.jayway.restassured</groupId>
             <artifactId>json-path</artifactId>
-            <version>2.5.0</version>
+            <version>2.8.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
```

Comment:
Only if I want to test json expressions like this:
```
response.andExpect(status().isOk())
    .andExpect(jsonPath('$.id').value(uuid as String))
```

I think that's a very good feature to have available in our test harness.